components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/dao/SAMLSSOServiceProviderConstants.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
```

Comment:
Fix licenses

Reply:
Resolved in [Introduce JDBC based persistence for SAML](https://github.com/wso2/carbon-identity-framework/pull/6241/commits/d1e67ce649f7a5d2e7e62f364ce93815f347e7b4)